src/support/timefuncs.c

```diff
@@ -29,30 +29,29 @@ extern "C" {
 #endif
 
 #if defined(_OS_WINDOWS_)
```

Comment:
doesn't matter much but the former is easier to add additional conditions to later if needed.